components/molecule/breadcrumb/src/index.js

```diff
@@ -20,17 +21,18 @@ const BreadcrumbBasic = ({
   return (
     <nav aria-label="breadcrumb" role="navigation">
       <div className={breadcrumbClassName({isExpanded, isScrollable})}>
-        <button onClick={expandBreadcrumb} className="sui-BreadcrumbBasic-btn">
+        {children && <div className={`${BASE_CLASS}-children`}>{children}</div>}
```

Comment:
maybe a more meaningful name instead of children? `leftElement` or `leftAddon` ?